src/views/base/navigation/header.module.scss

```diff
@@ -22,15 +22,22 @@ header {
 }
 
 .headerBorder {
+	background-color: var(--background_primary);
```

Comment:
I moved this since in some cases, the article's contents was overlapping with the header.